spk/dnscrypt-proxy/src/ui/cgi.go

```diff
@@ -239,14 +239,14 @@ func checkCmdExists(cmd string) bool {
 }
 
 // Execute generate-domains-blacklist.py to generate blacklist.txt
-func generateBlacklist () {
+func generateBlocklist() {
 	if !checkCmdExists("python") {
 		logError("Python could not be found or is not installed!")
 	}
 
 	var stderr bytes.Buffer
 	cmd := exec.Command("python", rootDir+"/var/generate-domains-blacklist.py")
```

Comment:
To rename...

Reply:
While I could change this one by renaming the file, I don't recommend it. This is the file name used from upstream:  https://github.com/jedisct1/dnscrypt-proxy/tree/master/utils/generate-domains-blacklists